Firestore/Source/Public/FIRQuery.h

```diff
@@ -256,6 +256,19 @@ NS_SWIFT_NAME(Query)
            isGreaterThanOrEqualTo:(id)value NS_SWIFT_NAME(whereField(_:isGreaterThanOrEqualTo:));
 // clang-format on
 
+/**
+ * Creates and returns a new `FIRQuery` with the additional filter that documents must
+ * satisfy the specified predicate.
+ *
+ * @param predicate The predicate the document must satisfy. Can be either comparison
+ *     or compound of comparison. In particular, block-based predicate is not supported.
+ *
+ * @return The created `FIRQuery`.
+ */
+// clang-format off
+- (FIRQuery *)queryFilteredUsingPredicate:(NSPredicate *)predicate NS_SWIFT_NAME(filter(using:));
```

Comment:
Shouldn't this `NS_SWIFT_NAME` be `filtered(using:)`? Mentioned this in the API proposal.

Reply:
Yes, changed.

Reply:
I'm really sorry to have pointed this out because I was totally wrong :-(.

@mikelehen points out that the other swift names are things like `orderBy` rather than `orderedBy` so this really should be `filter(using:)`.

Again I'm super sorry to have caused you extra work like this and I've reverted that change here.

Reply:
no problem, fellow. I like discussions, which is the only way great idea comes from.